config/CConfig.py

```diff
@@ -85,7 +90,7 @@ def __init__(self, sReferencePath="."):
         self.__dictConfig['sReferencePath'] = self.__sReferencePath # only to have the possibility to print out all values only with help of 'self.__dictConfig'
 
         # 1. basic setup stuff
-        self.__dictConfig['sPackageName']                = "robotframework-qconnect-base"
+        self.__dictConfig['sPackageName']                = "QConnectBase"
```

Comment:
Please refer to my comment https://github.com/test-fullautomation/robotframework-qconnect-base/pull/6#issuecomment-1062809012

Reply:
Hi Holger,
I have to apologize or the name-confusion. I corrected the comment.
The import mus be QConnectBase as it is now. I accidentaly wrote it wrong before.
Sorry for this additional work I produced.
Thank you,
 Thomas

Reply:
Hi Holger,
thank you, I merge now.
Thomas